src/icons/fileIcons.ts

```diff
@@ -48,7 +48,7 @@ export const fileIcons: FileIcons = {
       fileExtensions: ['sublime-project', 'sublime-workspace'],
     },
     { name: 'twine', fileExtensions: ['tw', 'twee'] },
-    { name: 'yaml', fileExtensions: ['yaml', 'YAML-tmLanguage', 'yml'] },
+    { name: 'yaml', fileExtensions: ['yaml', 'YAML-tmLanguage', 'yml', 'yml.dist'] },
```

Comment:
Yep, that looks good!

Reply:
Could you please add `yaml.dist` too so that we have consistency here?

Reply:
Okay 👍